.github/workflows/master.yaml

```diff
@@ -0,0 +1,157 @@
+name: publish
+
+on: push
+# TODO remove comment before merging
+#  branches:
+#    - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v1
+      with:
+        go-version: '1.13'
+    - name: Install kubebuilder
+      run: |
+        # from https://book.kubebuilder.io/quick-start.html#installation
+        os=`go env GOOS`
+        arch=`go env GOARCH`
+
+        # download kubebuilder and extract it to tmp
+        curl -sL https://go.kubebuilder.io/dl/2.0.0/${os}/${arch} | tar -xz -C /tmp/
+
+        # move to a long-term location and put it on your path
+        # (you'll need to set the KUBEBUILDER_ASSETS env var if you put it somewhere else)
+        sudo mv /tmp/kubebuilder_2.0.0_${os}_${arch} /usr/local/kubebuilder
+        echo "##[add-path]/usr/local/kubebuilder/bin"
+    - name: Install kustomize
+      run: |
+        opsys=`go env GOOS`
+        curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
+          grep browser_download |\
+          grep $opsys |\
+          cut -d '"' -f 4 |\
+          xargs curl -O -L
+        mv kustomize_*_${opsys}_amd64 kustomize
+        chmod u+x kustomize
+        sudo mv kustomize /usr/local/bin/
+    - name: Build & Test
+      run: make manager package test
+    - name: Check generated code is up to date
+      run: git diff --exit-code .
+    - name: Report coverage
+      run: bash <(curl -s https://codecov.io/bash)
+      env:
+        CODECOV_TOKEN: 39b82cdd-7ed3-4692-b2ea-f31ed9b84cef
+
+  publish:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.13'
+      - name: Install kubebuilder
+        run: |
+          # from https://book.kubebuilder.io/quick-start.html#installation
+          os=`go env GOOS`
+          arch=`go env GOARCH`
+
+          # download kubebuilder and extract it to tmp
+          curl -sL https://go.kubebuilder.io/dl/2.0.0/${os}/${arch} | tar -xz -C /tmp/
+
+          # move to a long-term location and put it on your path
+          # (you'll need to set the KUBEBUILDER_ASSETS env var if you put it somewhere else)
+          sudo mv /tmp/kubebuilder_2.0.0_${os}_${arch} /usr/local/kubebuilder
+          echo "##[add-path]/usr/local/kubebuilder/bin"
+      - name: Install kustomize
+        run: |
+          opsys=`go env GOOS`
+          curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases/latest |\
+            grep browser_download |\
+            grep $opsys |\
+            cut -d '"' -f 4 |\
+            xargs curl -O -L
+          mv kustomize_*_${opsys}_amd64 kustomize
+          chmod u+x kustomize
+          sudo mv kustomize /usr/local/bin/
+      - name: Package
+        run: make manager package
+      - name: gcloud auth
+        run: |
+          gcloud config set project cf-spring-pfs-eng
```

Comment:
is gcloud finally on the machine by default? 🕺

Reply:
yes!